bika/lims/browser/publish/emailview.py

```diff
@@ -510,7 +510,9 @@ def send_email(self, recipients, subject, body, attachments=None):
                                              attachments=attachments)
             sent = mailapi.send_email(mime_msg)
             if not sent:
-                logger.error("Could not send email to {}".format(pair))
+                msg = "Could not send email to {}".format(pair)
```

Comment:
I'm just wondering if this message also works for email, name pairs which contain unicode characters. Can you try that please with a recipient name that contains unicode characters?

Reply:
Thanks for the heads-up!. Just added this commit:
https://github.com/senaite/senaite.core/pull/1422/commits/077dd1378e275fe8eeaddd1a03ef22d17eaa00ac

And this is what I get when using pairs containing unicode chars:

![Captura de 2019-08-13 21-31-39](https://user-images.githubusercontent.com/832627/62971823-fe7ffc00-be12-11e9-9672-e21f29da4475.png)